spec/support/request_spec_helper.rb

```diff
@@ -0,0 +1,7 @@
+module RequestSpecHelper
+  #parse json response to ruby hash
```

Comment:
I think this comment should be `#parse ruby hash  response to json` since this method returns json